readme.md

```diff
@@ -68,12 +68,20 @@ python3 crawl.py -h
 ```
 ## Examples
 
-If you want to find the class whose name is 機器學習 in EECS and store the result to excel with filename <code>ML.xls</code>, you can use
+If you want to find the class whose name is 機器學�? in EECS and store the result to excel with filename <code>ML.xls</code>, you can use
 ```
-python3 crawl.py --search-opt Title=機器學習 --save ML.xls -b 9
+python3 crawl.py --search-opt Title=機器學�? --save ML.xls -b 9
 ```
 or maybe you just want to find the classes teached in Bioresources and Agriculture college, you can just type
 ```
 python3 crawl.py -b 6
 ```
+## The problem you probably will meet
```

Comment:
I think this can be merged into the environment part with some modification.

Reply:
I have deleted the part in the latest version